docs/design/proposals/non-graceful-node-shutdown.md

```diff
@@ -29,7 +29,20 @@ of service.
 > the taint prematurely may leave stale device states, active client sessions,
 > or lingering mounts, which can lead to serious data integrity issues.
 
-## Proposed solution
+## Proposed solution 2
```

Comment:
Lets keep the only working solution as the previous solution is not a valid solution anymore